src/MicrosoftTeams.ts

```diff
@@ -684,6 +684,26 @@ namespace microsoftTeams {
     sendMessageRequest(parentWindow, "openFilePreview", params);
   }
 
+  /**
+   * @private
+   * Hide from docs.
+   * ------
+   * Sideload an App-manifest directly to a team.
+   * This method works just for the first party Apps.
+   */
+  export function sideLoadAppManifestPackage(manifestBlob: Blob): void {
```

Comment:
We don't really use the term "sideloading" in any of our public-facing documentation. I think it might be better if we called this API something like "createCustomApp" or "uploadCustomApp".

Reply:
@aosolis, @MattSFT, thoughts?